index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+		<link rel="stylesheet" href="style.css">
+		<link rel="preconnect" href="https://fonts.googleapis.com">
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+		<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@300&family=Roboto:wght@100&display=swap" rel="stylesheet">
+		
+
+</head>
+<body>
+	<header id="ToolBar">
+		<a href="#"><img src="Images\Logo Placeholder.svg" alt="Logo" id="logo-placeholder"></a>
+
+		<a href="#" id="tool-Button">
+			<nav><img src="Images\MenuIcon.svg" alt="Menu Icon"></nav>
+		</a>
+	</header>
+
+		<main id="headline">
+			<section id="Introduction">
+				<div id="Name">
+					<h1>
+						Hey There. <br>
+						I’m Hamid.
+					</h1>
+
+				</div>
+				<div id="Status">
+					<h2>I'm a Software Developer</h2>
+				</div>
+			</section>
+				<div id="Service" >
+					<p>
+						I can help you build a product , feature <br>
+						or website Look through some of my <br>
+						work and experience! If you like what you <br>
+						see and have a project you need coded, <br>
+						don’t hesitate to contact me.
+					</p>
+					<ul id="Contacts">
+						<li><a href="#"><img src="Images/GithubIcon.svg"> </a></li>
+						<li><a href="#"><img src="Images/LinkedInIcon.svg"> </a></li>
+						<li><a href="#"><img src="Images/peace.svg"> </a></li>
+						<li></li><a href="#"><img src="Images/Twitter.svg"> </a></li>
+						<li><a href="#"><img src="Images/M.svg"> </a></li>
+					</ul>
+					<div id="bg-img">
+					
+					</div>
+					
```

Comment:
- [ ] As mentioned by the previous reviewer, kindly ensure that you remove the above `<div id="bg-img">...</div>` tag from your index.html file, then use the background image as `background-image: url(...)` in the container class of the Headline section.

Reply:
@tresorsawasawa I have done all the changes recommended, can you check again because I have my last review left. Can you check if there is anything else needed before I request my last review? Thanks